docs/useRedirect.md

```diff
@@ -46,3 +46,28 @@ redirect(false);
 ```
 
 Note that `useRedirect` allows redirection to an absolute URL outside the current React app.
+
+## `useNavigate`
```

Comment:
**polish**: I'd put this section in a tip but it's a personal opinion.

Reply:
done